Nuget package issue for the `Bswup` project

after running the stable nuget package action this error is throwing:

```
error NU5017: Cannot create a package that has no dependencies nor content. 
[D:\a\bitplatform\bitplatform\src\Tooling\Bit.Tooling.Bswup\Bit.Tooling.Bswup\Bit.Tooling.Bswup.csproj]
```

--- a/.github/workflows/nuget.org.yml
+++ b/.github/workflows/nuget.org.yml
@@ -29,7 +29,9 @@ jobs:
       
     - name: dotnet restore Bswup
       run: dotnet restore src/Tooling/Bit.Tooling.Bswup/Bit.Tooling.Bswup/Bit.Tooling.Bswup.csproj
-    - name: dotnet msbuild Bswup
+    - name: dotnet msbuild Bswup (1)
+      run: dotnet msbuild src/Tooling/Bit.Tooling.Bswup/Bit.Tooling.Bswup/Bit.Tooling.Bswup.csproj /P:Configuration=Release /p:GeneratePackageOnBuild=false
+    - name: dotnet msbuild Bswup (2)
       run: dotnet msbuild src/Tooling/Bit.Tooling.Bswup/Bit.Tooling.Bswup/Bit.Tooling.Bswup.csproj /P:Configuration=Release /p:GeneratePackageOnBuild=false
     - name: dotnet pack Bswup
       run: dotnet pack src/Tooling/Bit.Tooling.Bswup/Bit.Tooling.Bswup/Bit.Tooling.Bswup.csproj --output . --configuration Release

